examples/hif2a/generate_star_map.py

```diff
@@ -196,9 +196,11 @@ def mcs_star_map(threshold: float = 0.5):
     forcefield = Forcefield(ff_handlers)
 
     mols = []
+    # Ensure we pickle up mol properties
+    Chem.SetDefaultPickleProperties(Chem.PropertyPickleOptions.AllProps)
     for lig_path in map_config.ligands:
         supplier = Chem.SDMolSupplier(lig_path, removeHs=False)
-        mols.extend(list(supplier))
+        mols.extend([mol for mol in supplier])
```

Comment:
I'm fine either way, but out of curiosity, does the list(supplier) break?

Reply:
It appears to work on linux, but breaks on OSX. Will add a comment. The latter seems more explicit and seems to generally be safer when it comes to C++ wrapped code in my experience